src/View/Widget/MultiCheckboxWidget.php

```diff
@@ -210,7 +210,8 @@ protected function _renderInput($checkbox, $context)
             ];
 
             if ($checkbox['checked']) {
-                $labelAttrs = $this->_templates->addClass($labelAttrs, 'selected');
+                $selectedClass = $this->_templates->format('selectedClass', []);
```

Comment:
What happens when a user is using a custom set of templates that doesn't define `selectedClass`?

Reply:
I'm not sure it is relevant.
If the selectedClass is not included in custom set of templates, the stringTemplate will thrown an exception, like all missing required templates.

The selectedClass is set to `selected` by default in the `FormHelper`

Reply:
Wouldn't that template missing error cause applications to break when they update?

Reply:
@markstory He has added the new template in FormHelper, that's where all templates used by widgets are set.